src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -21,36 +47,421 @@ const ActorInit En_Bji_01_InitVars = {
+
+    static s16 sBlinkSequence[6] = { 0, 1, 2, 1, 0, 0 };
+    EnBji01* this = THIS;
+
+    s32 pad; /*pad until 4 main functions take gamestate that gets promoted to globalctx*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CCE98.asm")
+    this->actionFunc(this, globalCtx);
+    Actor_UpdateBgCheckInfo(globalCtx, (Actor*)this, 0.0f, 0.0f, 0.0f, 4U);
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CCEE8.asm")
+    if (this->blinkTimer-- <= 0) {
+        if (--this->blinkSeqIndex < 0) {
+            this->blinkSeqIndex = 4;
+            this->blinkTimer = (Rand_ZeroOne() * 60.0f) + 20.0f;
+        } else {
+            this->eyeTexIndex = sBlinkSequence[this->blinkSeqIndex];
+        }
+    }
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD028.asm")
+    Actor_SetHeight(&this->actor, 40.0f);
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD328.asm")
+s32 EnBji01_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD634.asm")
+    EnBji01* this = THIS;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD6B0.asm")
+    if ((limbIndex == 0) && ((globalCtx->gameplayFrames & 1) != 0)) {
+        *dList = NULL;
+    }
+    if (limbIndex == 0) {
+        rot->x = rot->x;
+        rot->y = rot->y;
+        rot->z = rot->z;
+    }
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD6C0.asm")
+    switch (limbIndex) {
+        case 8: /* Torso */
+            rot->x += this->torsoXRotAdj;
+            rot->z += this->torsoZRotAdj;
+            break;
+        case 15: /* Head */
+            rot->x += this->headXRotAdj;
+            rot->z += this->headZRotAdj;
+            break;
+    }
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD70C.asm")
+    return 0;
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/func_809CD77C.asm")
+void EnBji01_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Init.asm")
+    static Vec3f D_809CDCC8 = { 1088.0f, 1200.0f, 0.0f };
+    EnBji01* this = THIS;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Destroy.asm")
+    Vec3f sp20;
+    s32 temp_f4 = 0;
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Bji_01_0x809CCDE0/EnBji01_Update.asm")
+    if (limbIndex == 0xF) {
+        Math_Vec3f_Copy((Vec3f*)&sp20, &D_809CDCC8);
```

Comment:
sp20 is already a Vec3f, so &sp20 should be fine for this function

Reply:
Will be fixed in a batch commit.